### ISSUE TITLE
Write Model Generation Script

### ISSUE THREAD

**Issue by mathadoor** [mathadoor/ImageToLatex#4]
As the name suggests the model generation script will be used to create different types of models. The program should support the following:
1. Different types of CNN architectures to be used
2. Different types of RNN architectures to be used
3. Should be able to add anything new such as an attention-based mechanism if I need to.

**Comment by mathadoor** [comment]
The dataloader is currently good to go. I need to implement the model script and then the training script and we are good to go!

**Comment by mathadoor** [comment]
The forward function of the parser is working as expected. The next step is to build the loss function. The issue can be closed now.
